src/Maatwebsite/Excel/ExcelServiceProvider.php

```diff
@@ -56,6 +57,16 @@ public function boot()
 
         //Set the autosizing settings
         $this->setAutoSizingSettings();
+        
+        //Enable an "export" method on Eloquent collections. ie: model::all()->export('file');
+        Collection::macro('export', function($filename, $type = 'xlsx', $method = 'download') {
```

Comment:
In Laravel 5.0 Call to undefined method Illuminate\Support\Collection::macro()

Reply:
You could provide a PR to fix it ;)